assets/js/plugins/give-ajaxify-fields.js

```diff
@@ -0,0 +1,120 @@
+/*global $:false, jQuery:false */
+/**
+ * Note: This script is under development. We are using this inside core only and can be update in future.
+ */
+( function( $ ) {
+	'use strict';
+
+	$.extend({
+		giveAjaxifyFields: function( customSettings ) {
```

Comment:
@DevinWalker New plugin to handle ajaify fields like the country select option. currently, it supports only country state field